### ISSUE TITLE
website should have an `en` alias for English

### ISSUE THREAD

**Issue by ericzolf** [thundernest/thunderbird-website#287]
Entering https://thunderbird.net/ I get redirected to https://www.thunderbird.net/de/ but I wanted to download Thunderbird in English, so I entered https://thunderbird.net/en/ which didn't work. It took me a while to find out that `en-US` (or `en-GB`) was the right shortcut. As `English` points anyway to en-US, an alias of `en` would be consequent.

**Comment by Sancus** [comment]
Yeah, it is supposed to work that way. [These aliases](https://github.com/thundernest/thunderbird-website/blob/master/settings.py#L29) should work. Will get this fixed.

**Comment by MelissaAutumn** [comment]
Redirects will need to be made based on the aliases link since the wsgi server only runs on root. So if your accept language header has en, the redirect should work, but going to /en/ does not
